src/IssueViz/Editor/LocationTagging/LocationTagger.cs

```diff
@@ -172,8 +172,13 @@ private void TranslateTagSpans(ITextSnapshot newSnapshot)
             TagSpans = translatedTagSpans;
         }
 
-        private static bool OverlapsExists(Span span, NormalizedSnapshotSpanCollection spans) =>
-            spans.Any(x => x.OverlapsWith(span));
+        private static bool IntersectionExists(Span span, NormalizedSnapshotSpanCollection spans) =>
+            // We're using "IntersectsWith" rather than "OverlapsWith" so that zero-length spans are handled correctly.
+            // Given two spans A = [10 -> 15] and B = [12 -> 12],
+            // * A.IntersectsWith(B) == true
+            // * A.OverlapsWith(B) == false.
+            // This matters because when VS is requesting tags to show tooltips it uses a zero-length span.
```

Comment:
It uses a zero-length span?? Ohhh so that's why!

Reply:
Yep, and an empty span overlaps with nothing, even itself.  However, it will intersect.

Reply:
🤦 🤯